src/modules/esl-utils/dom/rect.ts

```diff
@@ -19,6 +19,15 @@ export class Rect {
     return new this(rect.x || rect.left, rect.y || rect.top, rect.width, rect.height);
   }
 
+  /**
+   * The static method checks the equality of the two Rect instances.
+   * @param rect1 - first instance of Rect
+   * @param rect2 - second instance of Rect
+   * */
```

Comment:
```suggestion
   */
```

Reply:
fixed in https://github.com/exadel-inc/esl/pull/711